VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html

```diff
@@ -157,21 +157,24 @@ <h1 class="govuk-heading-xl" [ngClass]="hearingSuspended() ? 'text-highlight-red
     <h2 class="govuk-heading-m" *ngIf="isNotStarted()">{{ 'judge-waiting-room.is-everyone-connected' | translate }}</h2>
 
     <p
-      class="govuk-body govuk-!-margin-bottom-3"
-      *ngIf="isNotStarted()"
-      [innerHTML]="'judge-waiting-room.hearing-messages.not-started' | translate: { phone: '0300 303 0655' }"
-    ></p>
+    class="govuk-body govuk-!-margin-bottom-3"
+    *ngIf="isNotStarted()"
+    [innerHTML]="'judge-waiting-room.hearing-messages.not-started' 
+      | translate: { phone: (hearingVenueIsScottish$ | async) ? contactDetails.scotland.phoneNumber : contactDetails.uk.phoneNumber }"
+  ></p>
 
     <p
       class="govuk-body govuk-!-margin-bottom-3"
       *ngIf="hearingPaused()"
-      [innerHTML]="'judge-waiting-room.hearing-messages.paused' | translate: { phone: '0300 303 0655' }"
+      [innerHTML]="'judge-waiting-room.hearing-messages.paused' 
+        | translate: { phone: (hearingVenueIsScottish$ | async) ? contactDetails.scotland.phoneNumber : contactDetails.uk.phoneNumber }"
     ></p>
 
     <p
       class="govuk-body govuk-!-margin-bottom-3"
       *ngIf="hearingSuspended()"
-      [innerHTML]="'judge-waiting-room.hearing-messages.suspended' | translate: { phone: '0300 303 0655' }"
+      [innerHTML]="'judge-waiting-room.hearing-messages.suspended' 
+        | translate: { phone: (hearingVenueIsScottish$ | async) ? contactDetails.scotland.phoneNumber : contactDetails.uk.phoneNumber }"
```

Comment:
We could reduce the repetition here by having the phone number as a variable in the code where we determine which one to use. The venue should never change during the lifetime of this component.

Reply:
@FarronF Sorry, could you elaborate a bit more? I'm a bit unclear

Reply:
Instead of having `(hearingVenueIsScottish$ | async) ? contactDetails.scotland.phoneNumber : contactDetails.uk.phoneNumber` multiple times. We could just have a phoneNumber that is set on init and use that as the parameter here. If we need it could also be a subject we could update based on if hearingVenueIsScottish$ updates but I don't think that's needed.

Reply:
@FarronF Both approaches would give us the desired outcome, but using async pipes on observables is regarded as best practice because we don't need to call subscribe on our observable, store the intermediate data and then unsubscribe.

Reply:
So we could have something like this:

declare the property in the waiting room base
`
phoneNumber$: Observable<string>;`
`

ngOnInit
`
this.phoneNumber$ = this.hearingVenueFlagsService.hearingVenueIsScottish$.pipe(
            map(x => (x ? this.contactDetails.scotland.phoneNumber : this.contactDetails.scotland.phoneNumber))
        );
`
and then wherever we need the phone number
`| translate: { phone: (phoneNumber$ | async) }"`

Reply:
@FarronF  The abstract base class is decorated as a directive, but it isn't used as one, therefore, the OnInit lifecycle hook wouldn't work as suggested. This would only work if the observable is initialized in the class constructor. I'm not a big fan of this but if you're happy for it to be done this way, then I'll make the change